.travis.yml

```diff
@@ -10,9 +10,11 @@ language: cpp
 # To experiment with Python 3, comment out Python 2.7 and uncomment one of the Python 3 versions.
```

Comment:
I don't mind switching the Python on Travis to Python 3 (assuming everything works). If so this comment can go.